cpp/log/logged_entry_test.cc

```diff
@@ -0,0 +1,7 @@
+#include "log/logged_entry.h"
+
+#include <gtest/gtest.h>
+
+typedef testing::Types<cert_trans::LoggedEntry> TestType;
+
+#include "log/logged_test-inl.h"
```

Comment:
Don't do this in that PR, though, it'll be messy. I'll do a quick one once you merged this...